tests/framework/installer/ceph_installer.go

```diff
@@ -505,20 +488,79 @@ func (h *CephInstaller) InstallRook() (bool, error) {
 		assert.True(h.T(), kerrors.IsNotFound(err))
 	}
 
-	// Create rook client
-	err = h.CreateRookToolbox(h.Manifests)
+	return true, nil
+}
+
+func (h *CephInstaller) InstallRook() (bool, error) {
+	if h.settings.RookVersion != VersionMaster {
+		// make sure we have the images from a previous release locally so the test doesn't hit a timeout
+		assert.NoError(h.T(), h.k8shelper.GetDockerImage("rook/ceph:"+h.settings.RookVersion))
+	}
+
+	assert.NoError(h.T(), h.k8shelper.GetDockerImage(h.settings.CephVersion.Image))
+
+	k8sversion := h.k8shelper.GetK8sServerVersion()
+
+	logger.Infof("Installing rook on K8s %s", k8sversion)
+	success, err := h.installRookOperator()
 	if err != nil {
-		return false, errors.Wrapf(err, "failed to install toolbox in cluster %s", h.settings.Namespace)
+		return false, err
+	}
+	if !success {
+		return false, nil
 	}
-	logger.Infof("installed rook operator and cluster %s on k8s %s", h.settings.Namespace, h.k8sVersion)
 
-	// disable admission controller test for Kubernetes version older than v1.16.0
-	if h.settings.EnableAdmissionController && !utils.IsPlatformOpenShift() && h.k8shelper.VersionAtLeast("v1.16.0") {
-		if !h.k8shelper.IsPodInExpectedState("rook-ceph-admission-controller", h.settings.OperatorNamespace, "Running") {
-			assert.Fail(h.T(), "admission controller is not running")
+	if h.settings.UseHelm {
+		err = h.CreateRookCephClusterViaHelm(map[string]interface{}{
+			"image": "rook/ceph:master",
+		})
+		if err != nil {
+			return false, errors.Wrap(err, "failed to install ceph cluster using Helm")
 		}
+	} else {
+		// Create rook cluster
+		err = h.CreateCephCluster()
+		if err != nil {
+			logger.Errorf("Cluster %q install failed. %v", h.settings.Namespace, err)
+			return false, err
+		}
+	}
+
+	logger.Info("Waiting for Rook Cluster")
+	if err := h.waitForCluster(); err != nil {
+		return false, err
+	}
+
+	if h.settings.UseHelm {
+		podName, err := h.waitForToolboxPod(h.settings.Namespace)
+		if err != nil {
+			return false, err
+		}
+		client.RunAllCephCommandsInToolboxPod = *podName
+	} else {
+		err = h.CreateRookToolbox(h.Manifests)
+		if err != nil {
+			return false, errors.Wrapf(err, "failed to install toolbox in cluster %s", h.settings.Namespace)
+		}
+	}
+
+	const loopCount = 20
+	for i := 0; i < loopCount; i++ {
+		_, err = client.ListPoolSummaries(h.k8shelper.MakeContext(), client.AdminClusterInfo(h.settings.Namespace))
```

Comment:
We are listing pool summaries as a simple command to verify ceph is responding in the toolbox? What about just a simple `ceph status` instead?